s2a/src/main/java/io/grpc/s2a/handshaker/ProtoUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2024 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.s2a.handshaker;
+
+import com.google.common.collect.ImmutableSet;
+
+/** Converts proto messages to Netty strings. */
+final class ProtoUtil {
+  /**
+   * Converts {@link Ciphersuite} to its {@link String} representation.
+   *
+   * @param ciphersuite the {@link Ciphersuite} to be converted.
+   * @return a {@link String} representing the ciphersuite.
+   * @throws AssertionError if the {@link Ciphersuite} is not one of the supported ciphersuites.
+   */
+  static String convertCiphersuite(Ciphersuite ciphersuite) {
```

Comment:
This looks like dead code.

Reply:
Done in https://github.com/grpc/grpc-java/pull/11540/commits/1e8485cd687738000169efea93e4c248fa4e5259